sdk/textanalytics/Azure.AI.TextAnalytics/perf/Azure.AI.TextAnalytics.Perf/Scenarios/DetectLanguagePerfTest.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Test.Perf;
+
+namespace Azure.AI.TextAnalytics.Perf
+{
+    public sealed class DetectLanguagePerfTest
+        : TextAnalyticsTest<PerfOptions>
+    {
+        private readonly TextAnalyticsClient _client;
+
+        private List<string> _batchDocuments;
+
+        public DetectLanguagePerfTest(PerfOptions options) : base(options)
+        {
+            // create client
+            _client = new TextAnalyticsClient(new Uri(TestEnvironment.Endpoint), new AzureKeyCredential(TestEnvironment.ApiKey));
+
+            // create input docs
+            var documentSize = 1000; // PR comment -> should be  'options.something()'
```

Comment:
@Mohit-Chakraborty 
should this value be hardcoded or should it come from PerfOptions as in java
if so, what is the attribute i should call?

Reply:
See my comment on this above. You can see the usages of `CountOptions` [here](https://adpnetsource.azurewebsites.net/#Azure.Test.Perf/CountOptions.cs,2d07a49ea3b0dbdf,references).